dataops-salesforce/README.md

```diff
@@ -51,4 +68,39 @@ For those who just want to use this as-is, here are the setup steps:
     
 7. Pull now and debug any issues
 
-Please contact your project team for managed service support provider for any issues or assistance debugging the setup or ongoing execution of this custom ingestion codebase
\ No newline at end of file
+Please contact your project team for managed service support provider for any issues or assistance debugging the setup or ongoing execution of this custom ingestion codebase
+
+### **DataOps Excel Plugin**
```

Comment:
This shouldn't be in this readme.

You should have your own readme in the new dataops-excel top level folder

Reply:
Hi Matt, I copied over and changed the folder structure to match what you recommended. One question I have is around files like dataops-excel/project/target/streams/compile/exportedProducts/_global/streams/export that has references to stuff on your machine - is there a way for me to compile to update those for the excel? Right now im using intelliJ as a staging area for development before copying it into databricks.

Reply:
Hey Reid,

We should be using the ,gitignore file to exclude any files from the target folder. I have examples in that for the salesforce project.

Anything in the target folder is generated by the compiler, and should not be pushed/stored in git. These files would be re-created by SBT once you open/compile the project.

Reply:
Annnnd I just realized that those files snuck into this repo with an early push - despite them being in the .gitignore file.

I'm going to add a commit to master to delete that folder from this repo.

Reply:
Okay - I just updated all the information in my PR